Doppler.Import.Subscribers.Cli/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Doppler.Import.Subscribers.Cli
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var serviceProvider = DefaultServiceCollection().BuildServiceProvider();
+
+            var configuration = DefaultConfiguration().AddCommandLine(args).Build();
+
+            Console.Out.WriteLine($"Environment: {configuration["ASPNETCORE_ENVIRONMENT"]}");
+            if(args.Length >= 1) Console.Out.WriteLine($"  Command line:");
```

Comment:
In general, we try to always use the curly braces.

Also, we use space between `if` and `(`

```suggestion
            if (args.Length >= 1) 
            {
                Console.Out.WriteLine($"  Command line:");
            }
```